Gems/PhysX/Code/Editor/ColliderComponentMode.h

```diff
@@ -46,6 +46,8 @@ namespace PhysX
         AzToolsFramework::ViewportUi::ClusterId GetClusterId() const override;
         AzToolsFramework::ViewportUi::ButtonId GetDimensionsButtonId() const override;
 
+        // ComponentMode ...
```

Comment:
uber nit: `ComponentMode overrides ...`

Reply:
Fixed, and I updated the others as well.